### PR TITLE
README: Add instructions for running unit tests

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+### Running internal unit tests
 ```sh
-$ export PATH=$PATH:$MPI_HOME/bin  
+$ ./autogen.sh
+$ ./contrib/configure-devel
+$ make
+$ make -C test/gtest test
+```
+
+### Running UCX performance test
```
Comment (miked-mellanox) [inline README]: need example how to run perftest w/o mpi as well (ssh)

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6986/ for details (Mellanox internal link).

File: README (hunk 1 above)
Comment (shamisp) [reply README]: +1 majority compiles without MPI. I would suggest to remove this MPI example all together. it is misleading !

Comment (miked-mellanox) [reply README]: maybe better to mark it as "optional" or for convenience of mpi users

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7001/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7003/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7004/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@shamisp pls take a look

**Comment by yosefe** [comment]
adding @gmegan

**Comment by shamisp** [comment]
👍

**Comment by gmegan** [comment]
+1

**Pull request merged**